src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c

```diff
@@ -241,20 +246,36 @@ static int zert_create_scu(struct zocl_ctrl_ert *zert, struct xgq_cmd_config_cu
 static void zert_init_cus(struct zocl_ctrl_ert *zert)
 {
 	u32 i = 0;
-	struct zocl_ctrl_ert_cu *cu = &zert->zce_cus[0];
+	struct zocl_ctrl_ert_cu *cu = NULL;
 
+	zert->zce_cus = kzalloc(sizeof(struct zocl_ctrl_ert_cu) * MAX_CUS,
```

Comment:
Please make it part of zert data structure since you know the size of array at compile time. The less memory allocation / free in driver, the better.

Reply:
Fixed